lib/firmata.js

```diff
@@ -1657,6 +1658,26 @@ Board.prototype.serialListen = function(portId) {
 };
 
 /**
+ * Update the baud rate of a configured serial port. Note that there is a 100ms delay required
+ * to update the baud in the firmware plus additional time to transmit this command. Wait a minimum
+ * of 200ms before writing to the serial port after calling this method.
+ * @param {number} portId The serial port to update the baud of.
+ * @param {number} newBaud The new baud rate to set.
+ */
+Board.prototype.serialBaud = function(portId, newBaud) {
```

Comment:
an alternate name could be `serialUpdateBaud` to avoid any confusion as to whether or not this sets the initial baud or is used only for updating the baud rate.